### DISCUSSION TITLE
Thank you for your work on this project

### DISCUSSION THREAD

**Discussion by j1mc** [ohmybash/oh-my-bash#389] [General]
I really appreciate the work people have done on this project.

Also, the README is the best README that I have read in some time.

Thank you very much. You are great. <3

**Comment by akinomyoga** [comment]
Thanks!

> Also, the README is the best README that I have read in some time.

Actually, this project is a fork of [ohmyzsh](https://github.com/ohmyzsh/ohmyzsh), and the README file originally came from that upstream project.

**Reply by akinomyoga** [reply]
Instead of just closing, I've converted the Issue into a Discussion.

**Comment by 0xis-cn** [comment]
Thanks too. After trying several shells in installations I found bash is exactly what I needed, and this project greatly simplified the configurations of bash.
